src/calva-fmt/src/config.ts

```diff
@@ -15,10 +15,9 @@ function configuration(
     cljfmtString: string
 ) {
     return {
-        'format-as-you-type': workspaceConfig.get('formatAsYouType') as boolean,
-        'keep-comment-forms-trail-paren-on-own-line?': workspaceConfig.get(
-            'keepCommentTrailParenOnOwnLine'
-        ) as boolean,
+        'format-as-you-type': workspaceConfig.get<boolean>('formatAsYouType'),
+        'keep-comment-forms-trail-paren-on-own-line?':
+            workspaceConfig.get<boolean>('keepCommentTrailParenOnOwnLine'),
```

Comment:
`get` is a generic function and so we can specify the type here without forcing the return type.